packages/core-mobile/app/services/send/SendServiceEVM.ts

```diff
@@ -41,8 +41,9 @@ export class SendServiceEVM implements SendServiceHelper {
         .executeAsync(async () => {
           const { amount, address, defaultMaxFeePerGas, token } = sendState
 
-          // Set canSubmit to false if token is not set
-          if (!token) return SendServiceEVM.getErrorState(sendState, '')
+          // Set canSubmit to false if token/address is not set
+          if (!token || !address)
+            return SendServiceEVM.getErrorState(sendState, '')
 
```

Comment:
when neither token/address has been entered, we don't have to show any error to users

Reply:
but do we show `address required` error when user entered address then remove it?